background/providers/kat.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+var https = require('https');
+var zlib = require('zlib');
+var path = require('path');
+var nconf = require('nconf');
+
+var log = require(__dirname + '/../logging.js');
+
+var Provider = require(__dirname + '/provider.js');
+
+var interval;
+
+class Kat extends Provider {
+    run() {
+        https.get('https://kat.cr/api/get_dump/hourly/?userhash=' + nconf.get('providers:kat:config:apiKey'), function (res) {
```

Comment:
This should also be using a value from the config.json since people may need to use a proxy address if kat.cr is blocked.